dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.messaging-3.0.internal.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.messaging-3.0.internal
```

Comment:
This is an internal feature common across each of the public messaging features, right?
Does there need to be a separate private API feature (`io.openliberty.jakarta.messaging.3.0`) for other components to pull in just the API ... or should they use the api bundle directly?

Reply:
Oddly no, not at this time.  Features that require the messaging api declare a dependency on this internal feature.  I can certainly add the API feature and modify the internal feature dependency.  Although the new API feature would be nearly vestigial, maintaining this feature pattern will facilitate serviceability and devOps.

Reply:
That would be fine if this feature only pulled in APIs needed for Messaging... but it also pulls in connectors internals and transactions.

Reply:
No problem!  I meant "maintaining the _api-specific_ feature pattern ...".  The change is uploaded.  Note that a feature specific to the javaee/javax jms api does not exist in previous releases.